lithops/serverless/backends/aws_lambda/config.py

```diff
@@ -65,74 +65,76 @@
+                        .format(type(config_data["aws_lambda"]["vpc"]["security_groups"])))
 
-    if not isinstance(config_data['aws_lambda']['efs'], list):
-        raise Exception("Unknown type {} for "
-                        "'aws_lambda/efs' section".format(type(config_data['aws_lambda']['vpc']['security_groups'])))
+    if not isinstance(config_data["aws_lambda"]["efs"], list):
+        raise Exception("Unknown type {} for \"aws_lambda/efs\" section"
+                        .format(type(config_data["aws_lambda"]["vpc"]["security_groups"])))
 
     if not all(
-            ['access_point' in efs_conf and 'mount_path' in efs_conf for efs_conf in config_data['aws_lambda']['efs']]):
-        raise Exception("List of 'access_point' and 'mount_path' mandatory in 'aws_lambda/efs section'")
+            ["access_point" in efs_conf and "mount_path" in efs_conf for efs_conf in config_data["aws_lambda"]["efs"]]):
+        raise Exception("List of \"access_point\" and \"mount_path\" mandatory in \"aws_lambda/efs section\"")
 
-    if not all([efs_conf['mount_path'].startswith('/mnt') for efs_conf in config_data['aws_lambda']['efs']]):
-        raise Exception("All mount paths must start with '/mnt' on 'aws_lambda/efs/*/mount_path' section")
+    if not all([efs_conf["mount_path"].startswith("/mnt") for efs_conf in config_data["aws_lambda"]["efs"]]):
+        raise Exception("All mount paths must start with \"/mnt\" on \"aws_lambda/efs/*/mount_path\" section")
 
     # Lambda runtime config
-    if config_data['aws_lambda']['ephemeral_storage'] < RUNTIME_TMP_SZ_MIN \
-            or config_data['aws_lambda']['ephemeral_storage'] > RUNTIME_TMP_SZ_MAX:
-        raise Exception(f'Ephemeral storage value must be between {RUNTIME_TMP_SZ_MIN} and {RUNTIME_TMP_SZ_MAX}')
+    if config_data["aws_lambda"]["ephemeral_storage"] < RUNTIME_TMP_SZ_MIN \
+            or config_data["aws_lambda"]["ephemeral_storage"] > RUNTIME_TMP_SZ_MAX:
+        raise Exception(f"Ephemeral storage value must be between {RUNTIME_TMP_SZ_MIN} and {RUNTIME_TMP_SZ_MAX}")
 
-    if 'region_name' in config_data['aws_lambda']:
-        config_data['aws_lambda']['region'] = config_data['aws_lambda'].pop('region_name')
+    if "region_name" in config_data["aws_lambda"]:
+        config_data["aws_lambda"]["region"] = config_data["aws_lambda"].pop("region_name")
 
-    if 'region' not in config_data['aws_lambda']:
-        raise Exception('"region" is mandatory under the "aws_lambda" or "aws" section of the configuration')
-    elif 'region' not in config_data['aws']:
-        config_data['aws']['region'] = config_data['aws_lambda']['region']
+    if "region" not in config_data["aws_lambda"]:
+        raise Exception("\"region\" is mandatory under the \"aws_lambda\" or \"aws\" section of the configuration")
+    elif "region" not in config_data["aws"]:
+        config_data["aws"]["region"] = config_data["aws_lambda"]["region"]
```

Comment:
What if region is set using options 1 (`~/.aws/config`) or 2 (env var)? Or is region mandatory in any case in the lithops config like in the documentation?
```yaml
lithops:
    backend: aws_lambda

aws_lambda:
    execution_role: <EXECUTION_ROLE_ARN>
    region: <REGION_NAME>
```

Reply:
Yes, I think we still need the region name for [this](https://github.com/lithops-cloud/lithops/blob/35599ecd198132c56fbdeaca36eb12164768f876/lithops/serverless/backends/aws_lambda/aws_lambda.py#L83C9-L83C17) or [this](https://github.com/lithops-cloud/lithops/blob/35599ecd198132c56fbdeaca36eb12164768f876/lithops/serverless/backends/aws_lambda/aws_lambda.py#L638C45-L638C45). In any case, the region from the Lithops config would override the region stated in the aws config.